subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/jvm/AbstractConsoleJvmTestWorkerFunctionalTest.groovy

```diff
@@ -48,8 +48,20 @@ abstract class AbstractConsoleJvmTestWorkerFunctionalTest extends AbstractIntegr
 
     def "shows test class execution #description test class name in work-in-progress area of console for single project build"() {
         given:
-        buildFile << testableJavaProject(testDependency(), MAX_WORKERS)
-        buildFile << testFrameworkConfiguration()
+        buildFile << """
+            plugins {
+                id 'java-library'
```

Comment:
The below test should be updated to use `java-library`. I forgot to update the plugins below. In general I try to update tests to use java-library if they apply `java`, since eventually we're going to have to do this for all our tests.